Where_Is_My_Wife/Assets/_Main/Scripts/Stage/LavacanonRight.cs

```diff
@@ -0,0 +1,32 @@
+using UnityEngine;
+
+public class LavacanonRight : MonoBehaviour
```

Comment:
Refactor the other one so only one cannon script exist, add a variable to specify the direction of shooting.